server/test/unit/server/webservices/test_urls.py

```diff
@@ -3,6 +3,40 @@
 from django.core.urlresolvers import resolve, Resolver404
 
 
+class TestDjangoUrls(unittest.TestCase):
+    """
+    Base class for testing Django Urls.
+    """
+    def assert_url_match(self, urls, url_name, match=True, *args):
+        """
+        Asserts that a url matches the url_name if match == True,
+        asserts that a url does not match the url_name if match==False.
+
+        :param url     : url or urls to match or base url to build from
```

Comment:
The param in the signature is 'urls' (with the s) but this says 'url' is the param name.